launch/WhiteRabbit - JavaFXUI.launch

```diff
@@ -7,7 +7,6 @@
         <listEntry value="1"/>
     </listAttribute>
     <listAttribute key="org.eclipse.debug.ui.favoriteGroups">
-        <listEntry value="org.eclipse.debug.ui.launchGroup.debug"/>

```

Comment:
Please revert this change. This removes the debug launch config favorite

Reply:
done